doc/docs/master/concepts/pipeline-concepts/datum/index.md

```diff
@@ -5,63 +5,64 @@
     all data at once, each directory independently, individual
     files one by one, or combined data from multiple inputs together.
 
-A datum is the smallest indivisible unit of computation within a
-job.
+## Definition
+A datum is **the smallest indivisible unit of computation within a
+job**.
 A job can have one, many or no datums. Each datum is processed
-independently with a single execution of the user code and
-then the results of all the datums are merged together to
+independently with a single execution of the user code on one of
+the pipeline worker nodes.
```

Comment:
`nodes` -> `pods`. This distinction is important because there can be multiple worker pods (kubernetes pods) on a single kubernetes node.